addons/account_payment/models/payment_transaction.py

```diff
@@ -89,14 +89,14 @@ def _compute_reference_prefix(self, provider_code, separator, **values):
                 return separator.join(invoices.mapped('name'))
         return super()._compute_reference_prefix(provider_code, separator, **values)
 
-    def _set_canceled(self, state_message=None):
+    def _set_canceled(self, state_message=None, extra_allowed_state=None):
```

Comment:
What if one of the provider has 2 extra allowed states to pass?

Reply:
CHanged into list, done :)

Reply:
Yes, it should be, I still forget to change account_payment sometimes.